### DISCUSSION TITLE
Finding ANLZ files of tracks

### DISCUSSION THREAD

**Discussion by tonelab77** [dylanljones/pyrekordbox#106] [Q&A] [answered]
Will this work to tell me what track the .DAT, .EXT, and .EX2 are associated with?

**Comment by dylanljones** [comment]
Hey,

Yes, that should work! You can get the path of the audio file the ANLZ files are associated with via the `PPTH` tag. All of the ANLZ files should contain this tag, for example

````python
from pyrekordbox.anlz import AnlzFile

anlz_path = "ANLZ0000.EXT"
anlz = AnlzFile.parse_file(anlz_path)
path = anlz.get("PPTH")
print(f"{anlz_path} is associated with audio file {path}")
````

**Reply by tonelab77** [reply]
I can't figure out how to run pyrekorbox. Been trying for hours, but it's still unclear. Also, I need to be able to find the ANLZ files by the track file name. Not the other way around. Is this possible?

**Reply by dylanljones** [reply, marked as answer]
What is causing issues?
The only thing that is kind of tricky (on Windows) is the SQLCipher dependency, but there are detailed instructions in the [documentation](https://pyrekordbox.readthedocs.io/en/latest/installation.html)
If you are having trouble finding the DB key, use the CLI as described [here](https://pyrekordbox.readthedocs.io/en/latest/key.html)

You can find the ANLZ files of tracks in the DB:
````python
from pyrekordbox import Rekordbox6Database

db = Rekordbox6Database()
content = db.get_content().first()  # Get a track from the database
path_dict = db.get_anlz_paths(content)
print("DAT-file: ", path_dict["DAT"])
````
